### PR TITLE
Cleaned up composer.json

### PR DESCRIPTION
**Pull request by webmozart**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,18 +20,22 @@
...
         "ramsey/uuid": "^2.8",
         "psr/log": "^1.0"
     },
     "require-dev": {
+        "webmozart/key-value-store": "^1.0-beta7",
         "predis/predis": "^1.0",
         "basho/riak": "^1.4",
-        "doctrine/cache": "^1.4",
         "phpunit/phpunit": "^4.6",
         "sebastian/version": "^1.0.1"
```
Comment (xabbuh) [inline composer.json]: This isn't needed neither, is it?

Comment (xabbuh) [inline composer.json]: But you seem to [require sebastian/comparator](https://github.com/puli/manager/blob/master/tests/bootstrap.php#L12).

Comment (webmozart) [inline composer.json]: That's a transitive dependency of PHPUnit. The explicit requirement of "sebastian/version" is there since 1.0.0 is buggy.

Comment (xabbuh) [inline composer.json]: I see, looks like PHPUnit should update the dependency then. Thanks for the clarification.

Comment (xabbuh) [inline composer.json]: However, I think you should `sebastian/comparator` explicitly though.

Comment (webmozart) [inline composer.json]: Could you open a PR?

Comment (webmozart) [inline composer.json]: Oh, I thought we were in a commit discussion. I'll add this here :)

Comment (xabbuh) [inline composer.json]: :) :+1:

**Pull request merged**